docs/src/pages/docs/crocks/Either.md

```diff
@@ -0,0 +1,1346 @@
+few. Defined as a tagged union type, it captures the essence of disjunction as
+it provides either a `Left` value or a `Right` value and not both.
+
+Unlike [`Maybe`][maybe], which fixes its "left side", or [`Nothing`][nothing] to
+a `()` (unit), `Either` is a functor in both it's `Left` and `Right` sides. This
+allows for the ability to vary the type on either side and is akin to the
+imperative `if...else` trees that are common in programming.
+
+Like most other types in `crocks`, `Either` has a right bias in regard to
+instance methods like `map`, `ap` and `chain`. This behavior can be used to
+formally capture Error handling as the `Left` value will be maintained and most
+the future operations will not be applied.
+
+```javascript
+import Either from 'crocks/Either'
+
+import compose from 'crocks/helpers/compose'
+import ifElse from 'crocks/logic/ifElse'
+import isNumber from 'crocks/predicates/isNumber'
+import map from 'crocks/pointfree/map'
+
+const { Left, Right } = Either
+
+// err :: a -> String
+const err = val =>
+  `${typeof val} is not an accepted type`
+
+// add :: Number -> Number -> Number
+const add =
+  x => y => x + y
+
+// validate :: a -> Either String Number
+const validate = ifElse(
+  isNumber,
+  Right,
+  compose(Left, err)
+)
+
+// flow :: a -> Either String Number
+const flow = compose(
+  map(add(10)),
+  validate
+)
+
+flow(32)
+//=> Right 42
+
+flow('32')
+//=> Left "string is not an accepted type"
+
+flow(true)
+//=> Left "boolean is not an accepted type"
+```
+
+<article id="topic-implements">
+
+## Implements
+`Setoid`, `Semigroup`, `Functor`, `Alt`, `Apply`, `Traversable`, `Chain`, `Applicative`, `Monad`
+
+</article>
+
+<article id="topic-construction">
+
+## Construction
+
+```haskell
+Either :: a -> Either c a
+```
+
+An `Either` is typically constructed by using one of the instance constructors
+provided on the `TypeRep`: [`Left`](#left) or [`Right`](#right). For consistency,
+an `Either` can be constructed using its `TypeRep` as a constructor. The
+constructor is a unary function that accepts any type `a` and will return a
+`Right` instance, wrapping the value it was passed.
+
+```javascript
+import Either from 'crocks/Either'
+import equals from 'crocks/pointfree/equals'
+
+Either(90)
+//=> Right 90
+
+Either.of(90)
+//=> Right 90
+
+Either.Right(90)
+//=> Right 90
+
+equals(
+  Either.Right([ 1, 2, 3 ]),
+  Either.of([ 1, 2, 3 ])
+)
+//=> true
+
+equals(
+  Either.of({ a: 100 }),
+  Either({ a: 100 })
+)
+//=> true
+```
+
+</article>
+
+<article id="topic-constructor">
+
+## Constructor Methods
+
+#### Left
+
+```haskell
+Either.Left :: c -> Either c a
+```
+
+Used to construct a `Left` instance of an `Either` that represents the
+`false` portion of a disjunction. The `Left` constructor takes a value of any
+type and returns a `Left` instance wrapping the value passed to the constructor.
+
+When an instance is a `Left`, most other methods `Either` returning methods on
```

Comment:
`most other methods Either returning methods` I think should be `other Either returning methods`